.dockerignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+videos
```

Comment:
Add this directory and anything else that's releveant in the `.gitignore` file

`+ image`

Reply:
Please comment on this